Sources/KlaviyoCore/AppContextInfo.swift

```diff
@@ -32,36 +32,36 @@ struct AppContextInfo {
 
     private static let deviceIdStoreKey = "_klaviyo_device_id"
 
-    let executable: String
-    let bundleId: String
-    let appVersion: String
-    let appBuild: String
-    let appName: String
-    let version: OperatingSystemVersion
-    let osName: String
-    let manufacturer: String
-    let deviceModel: String
-    let deviceId: String
-    let environment: String
+    public let executable: String
```

Comment:
We talked about this offline. Just make sure it's not exposed publicly outside the sdk. I think it's not!

Reply:
so looks these particular properties didn't need to be public but other similar public methods here are accessible outside. Will see how I can try and hide them if I can.